graph-builder/src/config/file.rs

```diff
@@ -73,6 +99,9 @@ impl MergeOptions<Option<UpstreamOptions>> for AppSettings {
     fn try_merge(&mut self, opts: Option<UpstreamOptions>) -> Fallible<()> {
         if let Some(upstream) = opts {
             self.try_merge(upstream.registry)?;
+            if upstream.pause_secs.is_some() {
```

Comment:
There are currently no requirements for Cincinnati releases (there aren't any really :) ). Do we want to keep it? I think its safe to remove it now, while we control the template and no disconnected installs are in the wild

Reply:
I will be very happy to remove it ASAP, but I'm hesitant to add the change to this PR. How about a separate ticket for rethinking the options?

Reply:
Do we have more deprecated options to remove though? I don't mind removing these later

Reply:
I would like to remove all `UpstreamOptions` but I didn't want to touch all of them in this PR. I touched this one now because it's not in the correct place. I added this option to the `ServiceOptions`.